paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/EmbeddedAppearanceSettingsDefinition.kt

```diff
@@ -0,0 +1,171 @@
+package com.stripe.android.paymentsheet.example.playground.settings
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+
+internal object RowStyleSettingsDefinition :
+    PlaygroundSettingDefinition<RowStyleEnum>,
```

Comment:
For embedded appearance, I think we'd actually just want to use our own data class, and serialize it as json directly using kotlinx.serialization, rather than needing the transformation layer.